examples/1-hello-world/index.js

```diff
@@ -0,0 +1,21 @@
+import { createServer } from "http";
+
+import {
+  createTextEvent,
+  createDoneEvent,
+} from "@copilot-extensions/preview-sdk";
+
+const server = createServer((request, ressponse) => {
+  console.log(`Received [${request.method}]`);
```

Comment:
I think we should log out both the method and the URL

```suggestion
  console.log(`Received ${request.method} ${request.url}`);
```

Reply:
I see you removed the brackets as well. It's a preference of mine to be able to differentiate between fixed text and variable text, and makes it really easy to see when a value is empty.

Reply:
we can leave in the brackets if you prefer. But values cannot be empty in this context

Reply:
Added